src/main/java/com/auth0/client/auth/AuthAPI.java

```diff
@@ -557,6 +559,22 @@ public AuthRequest login(String emailOrUsername, char[] password, String realm)
         return request;
     }
 
+    public AuthRequest mfaLogin(String mfaToken, String otp) {
```

Comment:
I'd use a different name for the method, in case we decide to implement login with the other challenge types in the future.

```suggestion
    public AuthRequest loginWithOTP(String mfaToken, String otp) {
```

Also, you should be checking for null parameters before trying to build a request that you can anticipate is going to fail. Check the other methods in the API client for reference.